halyard-deploy/src/main/java/com/netflix/spinnaker/halyard/deploy/spinnaker/v1/service/DistributedService.java

```diff
@@ -93,10 +93,10 @@ default Integer getLatestEnabledServiceVersion(AccountDeploymentDetails<A> detai
     enableDescription.put("requisiteStageRefIds", Collections.emptyList());
 
     Map<String, Object> disableDescription = new HashMap<>();
-    String disableId = "disable";
+    String disableId = "destroy";
```

Comment:
Yeah, from a usability perspective what happens is:

1. You deploy a new version/config
2. Something is wrong, so you rollback.
3. Fixing the bad server group requires redeploying.

Since you can't fix the newest disabled server group in place, and you (at least I cant imagine why) won't want to reenable that server group, there is no point it keeping it around. There's an argument to be made for keeping it around to debug, but that's far from what the average halyard user will want to do.